gameview.py

```diff
@@ -201,7 +201,7 @@ def on_update(self, delta_time):
                            self.shoot_pressed)
         self.check_collisions()
 
-        Enemy.update()
+        Enemy.update(delta_time)
```

Comment:
why is delta_time used here but not next row? should It be used every update function?

Reply:
Yup it should always be used, we should update the old outdated animations that do not make use of it